src/commands/dev/lib/dev-builder.ts

```diff
@@ -203,154 +179,36 @@ export async function executeBuild(
-            // subscription patterns, then this builder will be actually be
-            // executed.
-            devServer.output.debug(`Getting subscriptions for ${entry.fsPath}`);
-            const subscription = await builder.subscribe(buildConfig);
-            devServer.subscriptions.push({
-              type: 'Subscription',
-              patterns: subscription,
-              buildConfig: build,
-              buildEntry: entry,
-              buildTimestamp: Date.now()
-            });
-            continue;
-          } else {
-            devServer.output.debug(
-              `Building ${entry.fsPath} with "${build.use}" v${
-                builder[PACKAGE]!.version
-              } (workPath = ${workPath})`
-            );
-            output = await builder.build({ ...buildConfig, workPath });
-
-            if (typeof builder.prepareCache === 'function') {
-              const cachePath = getWorkPath();
-              await mkdirp(cachePath);
-              build.builderCachePromise = executePrepareCache(
-                devServer,
-                build,
-                {
-                  files,
-                  entrypoint,
-                  workPath,
-                  cachePath,
-                  config,
-                  meta: { isDev: true, requestPath: null }
-                }
-              );
-            }
-          }
-        } finally {
-          devServer.restoreOriginalEnv();
-        }
-
-        // enforce the lambda zip size soft watermark
-        const builderConfig = builder.config || {};
-        const { maxLambdaSize = '5mb' } = { ...builderConfig, ...config };
-        let maxLambdaBytes: number;
-        if (typeof maxLambdaSize === 'string') {
-          maxLambdaBytes = bytes(maxLambdaSize);
-        } else {
-          maxLambdaBytes = maxLambdaSize;
-        }
-
-        for (const asset of Object.values(output)) {
-          if (asset.type === 'Lambda') {
-            const size = asset.zipBuffer.length;
-            if (size > maxLambdaBytes) {
-              throw new LambdaSizeExceededError(size, maxLambdaBytes);
-            }
-          }
-
-          asset.buildConfig = build;
-          asset.buildEntry = entry;
-        }
+  cwd: string
+): Promise<BuildMatch[]> {
+  const matches: BuildMatch[] = [];
+  const builds = nowJson.builds || [{ src: '**', use: '@now/static' }];
+  for (const buildConfig of builds) {
+    let { src, use } = buildConfig;
+    if (src[0] === '/') {
+      // Remove a leading slash so that the globbing is relative to `cwd`
+      // instead of the root of the filesystem. This matches the behavior
+      // of Now deployments.
+      src = src.substring(1);
+    }
+    const entries = Object.values(await collectProjectFiles(src, cwd));
 
-        Object.assign(devServer.assets, output);
-      }
-    } catch (err) {
-      throw err;
+    for (const fileRef of entries) {
+      src = relative(cwd, fileRef.fsPath);
```

Comment:
So this `src` is not the same one from `buildConfig.src`, we might need to give it another name, and not to declare it as a `buildConfig` attr: https://github.com/zeit/now-cli/pull/2134/files#diff-d27ca1a3b1147a4215c6d0d96a08a4faR20

Reply:
It's the "resolved" source file (after the globbing), and there's one build match instance per globbed file. I don't think we need the original glob for anything so I don't see any harm in re-using `src`.

Reply:
I mean the `src` in the interface, not the local variable `src`. It cause me some confusion that I thought it was the `src` from `buildConfig`, according to `BuildMatch`'s type definition.